README.md

```diff
@@ -1,9 +1,9 @@
 # Logos
-Logos is a Perl regex-based preprocessor that simplifies the boilerplate code needed to create hooks for Objective-C methods and C functions with an elegant Objective-C-like syntax. It’s most commonly used along with the [Theos](https://theos.github.io/) build system, which was originally developed to create jailbreak tweaks. Logos was once integrated in the same Git repo as Theos, but now has been decoupled from Theos to its own repo.
+Logos is a Perl regex-based preprocessor that simplifies the boilerplate code needed to create hooks for Objective-C methods and C functions with an elegant Objective-C-like syntax. It’s most commonly used along with the [Theos](https://theos.dev/docs/) build system, which was originally developed to create jailbreak tweaks. Logos was once integrated in the same Git repo as Theos, but now has been decoupled from Theos to its own repo.
 
-Logos aims to provide an interface for [Cydia Substrate](https://cydiasubstrate.com/) by default, but can be configured to directly use the Objective-C runtime.
+Logos aims to provide an interface for [Cydia Substrate](http://cydiasubstrate.com/) by default, but can be configured to directly use the Objective-C runtime.
```

Comment:
I just [tweeted](https://twitter.com/_uroboro/status/1492976903631908867) Saurik about the https site not working (SSL cert expired) so hopefully he can fix it soon and you can revert this https→http change.

Reply:
Turns out it never had so no change is required